src/components/Onboarding/InitiateLink.js

```diff
@@ -92,8 +93,15 @@ class InitiateLink extends React.Component {
       approveLinkTxStatus,
       mkrApproveProxyTxHash,
       mkrApproveProxyTxStatus,
-      onComplete
+      onComplete,
+      toStep
     } = this.props;
+
+    if (!hotWallet) {
```

Comment:
2- if the user lands on the page with an activeAccount === coldWallet, our app state doesn't have access to hotWallet. It needs this in particular for account type.

4- One solution is to push them back to step one, but then they have to do onboarding over. In mycase the coldWallet was a metamask account, so I'd lose access to that vote proxy. 

Alternative solution would be to prompt the user to select their hot wallet in metamask & go from there.